miner/src/pool/verifier.rs

```diff
@@ -250,7 +250,13 @@ impl<C: Client> txpool::Verifier<Transaction> for Verifier<C, ::pool::scoring::N
 					return Err(err)
 				},
 			},
-			Transaction::Local(tx) => tx,
+			Transaction::Local(tx) => match self.client.verify_transaction_basic(&**tx) {
+				Ok(()) => tx,
+				Err(err) => {
+					info!(target: "txqueue", "[{:?}] Rejected local tx {:?}", hash, err);
```

Comment:
Why `info!` and not `error!`?

Reply:
Well, there is a distinction between log levels for client itself and what we want to report to user. `error!` usually means we will shut down the client because of internal error or something and here we just want to report to the user that the submitted tx was rejected.

Reply:
Fair enough.

Reply:
I'd go for a `warn`, as most other non-node-critical, but user-important messages are displayed like this, but with with it as is.